NGCHM/WebContent/javascript/DetailHeatMapDisplay.js

```diff
@@ -1552,8 +1553,37 @@ DET.setDetailDataHeight = function (mapItem, size) {
 	// Determine font size from CSS size of map, map fraction used by data view, and number of labels.
 	const numLabels = MMGR.isRow(axis) ? mapItem.dataPerCol : mapItem.dataPerRow;
 	const clientSize = MMGR.isRow (axis) ? mapItem.canvas.clientHeight : mapItem.canvas.clientWidth;
-	const skip = Math.floor(clientSize * mapFraction / numLabels) - 2;
-	return Math.min(skip, DET.maxLabelSize);
+	const skip = Math.floor(clientSize * mapFraction / numLabels); // Min space between labels.
+
+	const targetSize = Math.min (DET.maxLabelSize, Math.max (UTIL.minLabelSize, skip));
+	if (!fontSizeMap.has(targetSize)) {
```

Comment:
I don't know how much of an issue this is, and I don't know how often this is called, but the way it's written it "searches" the map for targetSize twice.